README.md

```diff
@@ -208,6 +208,7 @@
 - 💙🧩 [Tailwind UI](https://tailwindui.com) - Component library made with Tailwind CSS.
 - 💙📚 [Headless UI](https://github.com/tailwindlabs/headlessui) - Completely unstyled, fully accessible UI components.
 - 📚 [VueTailwind](https://github.com/alfonsobries/vue-tailwind) - Vue.js UI library using Tailwind CSS.
+- 📁 [EasyTailwind](https://easytailwind.now.sh) - Tailwind CSS templates handcrafted.
 - 📁 [Tailwind Made](https://tailwindmade.com/) - Paid, developer-friendly templates made with Tailwind CSS.
```

Comment:
```suggestion
- 📁 [Tailwind Made](https://tailwindmade.com/) - Paid, developer-friendly templates made with Tailwind CSS.
- 📁 [EasyTailwind](https://easytailwind.now.sh) - Tailwind CSS templates handcrafted.
```

Reply:
Can you put your item at below the previous one, and maybe make a more descriptive description?

Reply:
Will do